Source/SvgElement.cs

```diff
@@ -59,26 +58,30 @@ public void AddStyle(string name, string value, int specificity)
                 rules = new SortedDictionary<int, string>();
                 _styles[name] = rules;
             }
-            while (rules.ContainsKey(specificity)) specificity++;
+            while (rules.ContainsKey(specificity)) ++specificity;
             rules[specificity] = value;
         }
-        public void FlushStyles()
+
+        public void FlushStyles(bool children = false)

```

Comment:
Could you add documentation here, as this is a new public API?

Reply:
Thanks 👍 
Sorry for bothering you, but I would like a bit of information here _when_ to call this, e.g. why it is needed, as it is not as obvious as for example `AddStyle`.

Reply:
`SvgElement` originally has public `FlushStyles()` method.